modules/test/playwright/pages/portal-workflow-kaleo-designer-web/ProcessBuilderPage.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/ApplicationsMenuPage';
+
+export class ProcessBuilderPage {
+	readonly applicationsMenuPage: ApplicationsMenuPage;
+	readonly copyOfSingleAproverWorkflowDefinitionLink: Locator;
+	readonly page: Page;
+
+	constructor(page: Page) {
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+		this.copyOfSingleAproverWorkflowDefinitionLink = page.getByRole(
+			'link',
+			{
+				exact: true,
+				name: 'Copy of Single Approver',
+			}
+		);
+		this.page = page;
+	}
+
+	async clickCopyOfSingleAproverWorkflowDefinition() {
```

Comment:
As we talked, we should pass the randomized name to this function, so it selects and clicks it correctly. Like this:

```
	async clickWorkflowDefinition(name: string) {
		await this.page.getByRole('link', {
			exact: true,
			name,
		}).click();
	}
```
I have also renamed the method so any definition can be clicked by name.